src/main/java/pl/akademiaspecjalistowit/transactionalorder/product/ProductServiceImpl.java

```diff
@@ -40,6 +40,6 @@ private void removeBoughtOutProductsByName(String productName){
 
     @Override
     public void notifyOrderPlaced(OrderEntity orderEntityAfterValidations) {
-        removeBoughtOutProductsByName(orderEntityAfterValidations.getProduct().getName());
+        //removeBoughtOutProductsByName(orderEntityAfterValidations.getProduct().getName());
```

Comment:
Jak nie używane, to bardziej wywalić ;) Po to jest git, żeby dało się do czegoś wrócić w razie co. Nie przepadnie ;)